lib/rules/use-brace-expansion.js

```diff
@@ -28,15 +28,18 @@ module.exports = {
       CallExpression(node) {
         if (!ember.isComputedProp(node)) return;
 
+        const hasBraces = arr => !!arr.join('.').match(/\{.*\}/g);
```

Comment:
I had it like that at first, but then I was forced to do the `join` to each of the arrays inside the `if` predicate. I'm not strongly opinionated either way, but that's why I checked it in it this way.

Should I change it?